src/sagemaker/jumpstart/estimator.py

```diff
@@ -733,7 +733,7 @@ def attach(
         config_name = None
         if model_id is None:
 
-            model_id, model_version, config_name = get_model_id_version_from_training_job(
+            model_id, model_version, config_name = get_model_generic_info_from_training_job(
```

Comment:
nonblocking/nit: i would remove `generic` and just do `get_model_info...`

Reply:
sounds good